docs/source/Installation.rst

```diff
@@ -0,0 +1,62 @@
+Installation
+============
+
+SyReC is mainly developed as a C++ library.
+In order to make the tool as accessible as possible, it comes with an easy-to-use Python interface.
+
+We encourage installing SyReC via pip (preferably in a `virtual environment <https://docs.python.org/3/library/venv.html>`_):
+
+    .. code-block:: console
+
+        (venv) $ pip install mqt.syrec
+
+Once installed, start the SyReC editor GUI by running:
+
+    .. code-block:: console
+
+        (venv) $ syrec-editor
+
+In most practical cases (under 64-bit Linux, MacOS incl. Apple Silicon), this requires no compilation and merely downloads and installs a platform-specific pre-built wheel.
+
+.. warning::
+        Currently the FLINT2 dependency is not automatically installed. Thus, windows users need to install FLINT2 by hand.
+        Either by building `from source <http://www.flintlib.org/doc/building.html>`_ or downloading the corresponding `.zip file <http://www.flintlib.org/downloads.html>`_.
+        We are working towards fixing this issue.
+
+.. note::
+    In order to set up a virtual environment, you can use the following commands:
+
+    .. code-block:: console
+
+        $ python3 -m venv venv
+        $ source venv/bin/activate
+
+    If you are using Windows, you can use the following commands instead:
+
+    .. code-block:: console
+
+        $ python3 -m venv venv
+        $ venv\Scripts\activate.bat
+
+    It is recommended to make sure that you are using the latest version of pip, setuptools, and wheel before trying to install the project:
+
+    .. code-block:: console
+
+        (venv) $ pip install --upgrade pip setuptools wheel
+
+Building from Source for Performance
+####################################
+
+In order to get the best performance out of SyReC and enable platform-specific compiler optimizations that cannot be enabled on portable wheels, it is recommended to build the package from source via:
+
+    .. code-block:: console
+
+        (venv) $ pip install mqt.syrec --no-binary mqt.syrec
+
+This requires a `C++ compiler <https://en.wikipedia.org/wiki/List_of_compilers#C++_compilers>`_ compiler supporting *C++17* and a minimum `CMake <https://cmake.org/>`_ version of *3.14*.
```

Comment:
This is missing the boost requirement.

Reply:
Also, the minimum version required is 1.71.0 right?

Reply:
See above ☝️.

Reply:
done 👍🏻